test/test_benchmark.py

```diff
@@ -399,7 +399,7 @@ def get_output_files(self):
     ]
   else:
     benchmarkers += [
-      # EmscriptenBenchmarker('Node.js', config.NODE_JS),
+      EmscriptenBenchmarker('Node.js', config.NODE_JS),
```

Comment:
There seem to be a lot of commented out configurations.   I do wonder what we should do by default here.  When I want to measure something I normally edit this file to enable just the one or two cases I want to compare.   Do other folks also do this?  @kripken ?   Should we have some way to select on the command line which configurations we want to run?